index.html

```diff
@@ -586,5 +593,79 @@ <h3 class="text-[20px] font-medium text-black">
       </div>
     </section>
     <!--* END ABOUT MYSELF -->
+
+    <!--* footer -->
+    <footer
+      class="flex flex-col justify-between items-center bg-light-blue w-full overflow-hidden px-5 rounded-tl-[90px] before:content-[''] before:h-[700px] before:w-[65%] before:bg-mobile-footer-shapes before:bg-contain before:-z-1 before:absolute before:left-[35%] before:bg-right before:bg-no-repeat before:mt-5"
```

Comment:
- [ ] No comment on this section, your code follows the project requirements, but we have a few. things to modify here before we move forward, it is about the background image, please take a look at this image below and fix the change
- [ ] The second thing is your text alignment, please see how it is adjusted in the original design and make that change @martinkarugaba 

<img width="735" alt="fig" src="https://user-images.githubusercontent.com/77843050/206512402-ab88bca1-2cc9-4221-bd8e-2777cab5b0fc.png">